sentences/en/_common.yaml

```diff
@@ -96,13 +96,14 @@ lists:
 expansion_rules:
   name: "[the] {name}"
   area: "[the] {area}"
-  what_is: "(what's | whats | what is)"
-  brightness: "{brightness}[%| percent]"
-  turn: "(turn | switch | change)"
-  temp: "(temp | temperature)"
-  temperature: "{temperature}[°| degrees] [{temperature_unit}]"
+  what_is: "(what's|whats|what is)"
+  brightness: "{brightness}[%|percent]"
+  light: "[the] (light|lights|lighting)"
+  turn: "(turn|switch|change)"
+  temp: "(temp|temperature)"
+  temperature: "{temperature}[°|degrees] [{temperature_unit}]"
```

Comment:
Same thing here without the space. `20degrees` is maybe a bit unnatural.

Reply:
Done, also added an optional space before the unit